doc/api/v1.yaml

```diff
@@ -170,10 +170,15 @@ paths:
                               type: object
                               required:
                                 - reason
+                                - date
                               properties:
                                 reason:
                                   description: Reason for rejection
                                   type: string
+                                date:
+                                  description: Epoch and slot ID of block where the node failed to apply the fragment
+                                  type: string
+                                  pattern: "[0-9]+\\.[0-9]+"
```

Comment:
I was not quite sure of including that in the api, but given that we already have that info I figured out it might be useful somehow. 
Actually, given that we reset the logs after a branch switch it does uniquely identify that block in the chain that is being used by the node, and it may also be easier to use than a single hash, for which you would have to search to which block it corresponds.

Reply:
We could also add the block hash like it's done for `InABlock` or remove it from the public api and only use that internally

Reply:
I'd say we should remove this from the API for now, until we have a better idea what to use it for.
This also nicely sidesteps the awkwardness of extending the "legacy" `/v0` API.